app/assets/javascripts/registration/register.js.erb

```diff
@@ -1,4 +1,5 @@
 $.ajaxQ('register', { maxRequests: 10 });
+$('#external_workflow_dropdown').hide(); // external (e.g. goobiWF) workflow selector hidden by default
```

Comment:
Should we hide this by default using a css class or something in the HTML to prevent possibly displaying the field briefly?